frontend/lib/src/components/elements/DeckGlJsonChart/useDeckGl.tsx

```diff
@@ -234,18 +233,6 @@ export const useDeckGl = (props: UseDeckGlProps): UseDeckGlShape => {
       }-v9`
     }
 
-    // Set width and height based on the fullscreen state
-    if (isFullScreen) {
-      Object.assign(copy.initialViewState, { width, height })
-    } else {
-      if (!copy.initialViewState.height) {
-        copy.initialViewState.height = DEFAULT_DECK_GL_HEIGHT
-      }
-      if (shouldUseContainerWidth) {
-        copy.initialViewState.width = width
-      }
-    }
```

Comment:
It looks like this just does not work. My guess is that before it was typed as `any` so we never caught the issue.

https://github.com/visgl/deck.gl/blob/v8.9.36/modules/core/src/lib/deck.ts#L162

Nonetheless, it seems like everything is okay from testing.

https://github.com/user-attachments/assets/0685317d-cc7f-45f8-8c98-8d1e77ca8cfa

Reply:
Thanks for the video here. I see [the underlying type](https://github.com/visgl/deck.gl/blob/v8.9.36/modules/core/src/lib/deck.ts#L158-L162) was indeed `any`, so everything you wrote here makes sense to me!